pkg/cluster/internal/kubeconfig/kubeconfig.go

```diff
@@ -32,8 +32,8 @@ import (
 
 // Export exports the kubeconfig given the cluster context and a path to write it to
 // This will always be an external kubeconfig
-func Export(p providers.Provider, name, explicitPath string) error {
-	cfg, err := get(p, name, true)
+func Export(p providers.Provider, name, explicitPath string, external bool) error {
```

Comment:
nit: now that  we are using`internal` more widely, might be worth refactoring the boolean arguments to mean `internal` instead of `external` in all the places for consistency.

Reply:
It is really not obvious, why in the middle of the path, internal becomes !external. I just kept it the way it was, so that my (first) PR has the smallest possible size.

As there may be some future work related to this code (#2246, discussion #2217), this may be reworked anyway.

Please let me know if there is anything for me to left do to.